sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -767,6 +771,7 @@ async def update_rule(
             )
         )
         request_body = create_entity_body.serialize(is_xml=True)
+        serialize_rule_key_values(request_body, rule)
```

Comment:
Double checking: It's intentional that we return the successfully set rule from create, but not from update?

Reply:
Python's design was to return None because "update" could return a XXXDescription with a partial set of attributes.
There was be a broader discussion about update_xxx with etag. I'll cross-check with other languages.
But for this PR, let's not touch this this.